src/masternodes/rpc_poolpair.cpp

```diff
@@ -1026,6 +1038,195 @@ UniValue testpoolswap(const JSONRPCRequest& request) {
     return UniValue(res.msg);
 }
 
+
+UniValue testcompositeswap(const JSONRPCRequest& request) {
+    auto pwallet = GetWallet(request);
+
+    RPCHelpMan{"testcompositeswap",
+               "\nTests a compositeswap transaction with given metadata and returns compositeswap result.\n" +
+               HelpRequiringPassphrase(pwallet) + "\n",
+               {
+                       {"metadata", RPCArg::Type::OBJ, RPCArg::Optional::NO, "",
+                        {
+                                {"from", RPCArg::Type::STR, RPCArg::Optional::NO,
+                                 "Address of the owner of tokenA."},
+                                {"tokenFrom", RPCArg::Type::STR, RPCArg::Optional::NO,
+                                 "One of the keys may be specified (id/symbol)"},
+                                {"amountFrom", RPCArg::Type::NUM, RPCArg::Optional::NO,
+                                 "tokenFrom coins amount"},
+                                {"to", RPCArg::Type::STR, RPCArg::Optional::NO,
+                                 "Address of the owner of tokenB."},
+                                {"tokenTo", RPCArg::Type::STR, RPCArg::Optional::NO,
+                                 "One of the keys may be specified (id/symbol)"},
+                                {"maxPrice", RPCArg::Type::NUM, RPCArg::Optional::OMITTED,
+                                 "Maximum acceptable price"},
+                        },
+                       },
+                       {"inputs", RPCArg::Type::ARR, RPCArg::Optional::OMITTED_NAMED_ARG, "A json array of json objects",
+                        {
+                                {"", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED, "",
+                                 {
+                                         {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The transaction id"},
+                                         {"vout", RPCArg::Type::NUM, RPCArg::Optional::NO, "The output number"},
+                                 },
+                                },
+                        },
+                       },
+               },
+               RPCResult{
+                       "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
```

Comment:
Incorrect result. Please update the return help test for compositeswap and estimatecompositepaths as well.

Reply:
:+1: